src/data/abilities.h

```diff
@@ -1444,41 +1444,41 @@ const struct Ability gAbilitiesInfo[ABILITIES_COUNT] =
     [ABILITY_DESOLATE_LAND] =
     {
     #if B_EXPANDED_ABILITY_NAMES == TRUE
-        .name = _("Desolate Land"),
+        .name = _("DESOLATE LAND"),
     #else
-        .name = _("DesolateLand"),
+        .name = _("DESOLATELAND"),
     #endif
         .description = COMPOUND_STRING("Summons intense sunlight."),
         .aiRating = 10,
     },
 
     [ABILITY_DELTA_STREAM] =
     {
-        .name = _("Delta Stream"),
+        .name = _("DELTA STREAM"),
         .description = COMPOUND_STRING("Summons strong winds."),
         .aiRating = 10,
     },
 
     [ABILITY_STAMINA] =
     {
-        .name = _("Stamina"),
+        .name = _("STAMINA"),
         .description = COMPOUND_STRING("Boosts Defense when hit."),
         .aiRating = 6,
     },
 
     [ABILITY_WIMP_OUT] =
     {
-        .name = _("Wimp Out"),
+        .name = _("WIMP OUT"),
         .description = COMPOUND_STRING("Flees at half HP."),
         .aiRating = 3,
     },
 
     [ABILITY_EMERGENCY_EXIT] =
     {
     #if B_EXPANDED_ABILITY_NAMES == TRUE
-        .name = _("Emergency Exit"),
+        .name = _("EMERGENCY EXIT"),
     #else
-        .name = _("EmergncyExit"),
+        .name = _("EMERGNCYEXIT"),
```

Comment:
For the examples where we've run two words together without a space, won't they decap incorrectly (like the PokéNav in #4181). e.g. here will we get "Emergncyexit"?

Reply:
Related: #4241 introduces spaces into all the names by default (but this problem will still exist if the official names are turned off).